src/Api/Response/Result/BaseItem.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Findologic\Api\Response\Result;
+
+class BaseItem
```

Comment:
Can you please tell me, why we need this classes within the plugin? Those already exist within `findologic-api`.

https://github.com/findologic/findologic-api/blob/develop/src/Responses/Json10/Properties/BaseItem.php

Reply:
Cannot use a library inside the plugin structure